src/GitExtensions.Extensibility.Tools/tools/Download-GitExtensions.ps1

```diff
@@ -0,0 +1,122 @@
+param([string] $ExtractRootPath, [string] $Version, [string] $Source = "github")

```

Comment:
```suggestion
param(
  [Parameter(Mandatory=$true)]
  [string] $ExtractRootPath, 
  [Parameter(Mandatory=$true)]
  [string] $Version, 
  [ValidateSet('GitHub','AppVeyor', ignorecase=$False)]
  [string] $Source = "GitHub")
```

This way you can remove checks on lines 3-16.
NB: I have just discovered `ValidateSet`, so I've never actually used it.

See also:
* https://devblogs.microsoft.com/scripting/use-powershell-to-make-mandatory-parameters/
* https://www.gngrninja.com/script-ninja/2017/1/21/powershell-quick-tip-using-validateset
* https://learn-powershell.net/2014/02/04/using-powershell-parameter-validation-to-make-your-day-easier/

Reply:
What is better, `ValidateSet` or `enum` suggested in [original PR](https://github.com/gitextensions/gitextensions.extensibility/pull/7#discussion_r290074103)?

Reply:
I'd go for `ValidateSet`, if it works.
My experience with enums in PS is not that great...

Reply:
Ok. I found that `ValidateSet` even completes values on command line.